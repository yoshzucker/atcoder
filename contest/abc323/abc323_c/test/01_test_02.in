100 100
1800 1800 2100 2100 1700 700 1600 1100 2300 700 700 1600 1200 1700 1800 1900 2000 1700 2200 1500 800 1600 1900 1700 2300 1300 500 2200 2400 2100 1900 1100 1600 2500 1300 1000 2500 2000 1500 2000 1000 1800 1200 1900 800 500 2000 1500 500 2000 900 1900 1100 1500 1700 1000 500 1500 800 500 700 2100 1100 1800 800 2100 2200 1700 600 900 1700 2200 2400 1000 600 1200 1500 1200 2100 1100 1600 900 700 600 1200 2500 1400 2200 500 2300 1500 800 1800 1300 1000 1700 800 600 2400 700
oooxooxooxxxoxxooxoxxoxoxoxxooooooxooxxxoxoxxooooooooxoxxoxxooxxoxxoxooxooooxxoxoxoooxooxxooxxooxxoo
xoxxoxooxoxxoxoxxxxooxooxooooxoxxxxxoooooxxxxxxoxxooxoxxoxxxoooxooxxxxoxooxoxxxxoxoooooxxoxxxxxooxxx
xxoxoxooxxxoxxxoxooxxxoxxoxxxoxxxxoooooxoxooxxxxxxxoxoxooxoxoooxxxoxooxooxoooxxoxxxxxxxoxxxoooooxxxx
xxooxxxxxooxxxxoxoooxoxoxoooxoxooxoxoxxxooooxxxxooxooxooxoxoxoooxxxxooxoxooooxoxooxoooxxxoxoxoxoooxx
xxoooxooxoxooooooxoxoooooxxxxoxxxoooxxoooxoxxoooxoxoooxooxoxoooxxooxooxxxxxxxxxoxxxxxxooooxxxxoxxxxx
oxoxxooooooxxoxxoxoooxoxxxxxxxxxxxooxoxooxoxxoxooooxoxxxoooxxxoxoooxxxoxooxoxxxoxxxxooooooxoxoxxxoxx
xoxoxooxxoooxxoxooxxooxxxooxoxooooxooooxxoxoxooxooooxxxooxxxooxoxxxxxoxoxxoxoooxxooxxxxxxooxooxoxoxo
ooxxoxxooooxxxxooxxxooxoxooxxxxooxooooxxxxoxxoxxxoooxxooxooxoooxxoxxxxxooxxxooxoxoxxxxooooooxooxoxxx
oxoxxxxooooooxoxoxxooxxxoooxoooxooxoxxxoxxoxoxxoxooooxxxxxxxxoxoxxxxxxooxooxoxxoxooxxxxoxxxxxoxxoxxo
xoooxxxxxoxooxxoxoxoxooxoooxxoxoxxxoooxoxoxoooxoxoxoxoxxxooxooooooooooooxoxxoooxoxoxooxxxxxxxoooxxxo
oooxxxooxxxxooxxoxooxoooooxoxoooxoooxxxoxxxoxxooxxooxxooxooxxxxxxoxooooooxxxxoxxoxooxoxxxooooooxoxxx
xoxxxoxxxxoxoxxxooooooxxxoxxoxxxooxoooxoooxoxxxxooooooxooxxooxxxxxxxxxoxoxooxoooooxxoxxoxxxoxoxooxxo
oooooxooxxxxxoxoxxooxxxxooooxxxoxooxooxxoxxoxxoxxxooxoxxxxxooxoxoxoooxxxoooxooxoxxxoxooxooxooxxoooxx
oxxoxooooxxoxxoxxxooooooxxoooxoooooxoxoxooooxxoxooooxxoxxxxxoooooooxooooxoxxoxxxxoxooxxoxxxxoooxxooo
xooooxxxxxxooxxxxxoxooxxxoxxxxooxxoxxoooxoxoxooxoxooooxoxoxxxxooxooooooooooxxxxoxoxoxoxoxoxoxoxxoxoo
oxxoooooxooooxoooooxoxooxooxooxxxxoxxxooxoxoxxxoooxoooooooxoxooxoxoxoxoxxooxxxooxooxxxxxxxoxooxxooxo
oooxoxxxooxoooxooooxxoxxoxoxxooxxoxxxooxxxooxoxxooxooxooxxoxxxoxooooooooooxooxxoxoooooxoxooxxoxoxxxo
xoxoxxxxoxxxxoxoxoxoxooxxoxxoxoxxooxoxoxoxxxxooxoxoxoooxxxoooxoooxoxxxoooxooxxxxoooxoxxoxxxxooooxxxx
oxoxoxxxooxxxoooxxxoxxxoxooxxoxooxxxxooxxooxoxxxoxxoxooxoxoxoxxoxoxooxxoxoxxoxxxoxoxooxoooxxoxooooxo
oxooxxxxxxooxxoxoxxooxoxoooxooxoooxooooxxxxoxoxooooxooooxoxxxxxxxooxxxoxoxxoxxooxooxxxoxxoooxxxxoxxx
oxxoxxoxoxooxxxxxoxxoxxoooxoxoxoxoooxxxxooxxoxoooxoxxxooxxxxxoxxxooxoxooxoxoxxooxxxoxxxxxxoxoxxoooxo
xoxxoxxxooxoxoxxoxoxooooooxxxxoooooxooooxooooxoooxxxxoxxooooxxoxxoxoxooooxooxxoxooxoxoooxxxxooxooxoo
xxxxxoxxoooxxxoooooxxxoooxxoxxooxoooooxoxoxxxxxxoxxxooxooxxxooxxoxxooxoooxxxoxxxxooxxxxoxoxxoxxxooxx
xxxoxooxxooxoooxxxxoxoxoxoxoooxxoxxxooxoxxoooooxooxoooxxoooxoooooooooxxxooooxxxoxxxxxxxxxooxoxxxxoxx
xoooooxooooxoxoxoxxooxxoxxxxxxoxxooxoxxxoxxoxoxxoooxooooooxxxoxoxxoxxxoxxoxxxooooxooooooxoxxxooooxxo
xxooxoooxxxoxxoxxoxxooxxxxxoooxxooooxoxxxoxxooxoxooxoxooxxoxoxoxxoxoxoooxoxoooxxooxooxxoxxoxoxxooxox
xoxooxxxoooooxoxoxooxoxxoxxoooxxxxxxxoxxoxoooxoxoxxooooxxxoooooxxxxxxxoxoxoxxxooxxxxoxxxoxxxoxooxxox
xoxoxoxxxoxooooooooooooxooxooooooxoxxxoxxoxoxxxoxoxxxoxooxoxooxoxoxxxxxooxoxoxxoxxoxxoxxoooooxxxoxxx
ooooxxoxoooxoxxoooxxooxxooxooxxxoxooxxoooooxxxxoxoxoooxoooooxoxxxoxxxoxxoxoxoxoxxoxoxxxxoxoxxxxooxox
xxxxooxoxoxooooooxoooxoxxooooxxoooooxoxooxooxoxxxooxoxoxoxxoxxoooxxooxoxxoxxoxoxooxxxxoooooxxxxooxxo
xoxooxoxxxxoooxoxoooxxxxxxoxoooxoxxxoxxxxxoxoxxxxxooooxoxoxxoxxxooooxxooxoxooxooxxooooxxoxxxoxooooox
oxooooooooxxoxxoxxxxxooxooxxoooooxxoxxxoxxooxoxxooxoxxxxooxxoxoxxoooxxxooxxxxxooxxooxxxxxoxooxxxxxxo
xxxoxoxoxoxxooxooxoxxoooooooxxooxoxooxoooxxooxooxxoxooxooooxxxxoooooooxxoxooooxooxooxoxxoooxxooooooo
oxxooxxooxxoxoooxoxxxxxoxxxxxxxooxoxxxoooxxoxxoxooxooxooxoxxoxoxoxooxxxoooxooxooxxoxxxxoxxooooooxxox
ooooooxxxxooxxoxoxoooxxxxxxxxxooooxooooxxooxoxxooxoooxoxooxoxxxxooooxxxxoxxooooooooooxxoxxxooxoxxxox
xxooooxoxooooxooxxoxoooooxooxxxxoxxooxxooxxxxxooxoxoxxooxooxxoxoxoxxxxoxoooxxoxxoxoxooxoooxooxxxxoxo
xxooxoxoooxxxooxxooxoxoxoxoxoxxooxxxoxoxxxxxoxxxxxoxxoxoxooooxxxxoxxxxoxxoooxxoxooxxxooxxooooxoxooxo
oooooooooxxxooxoxxooxoxoxoxxoxoxxxoxoxxxxxoxoxxooxxooooxxooxoooooxoooooooxxxxxoxoxoxxooxooxxxooxxxox
xxoxoooxxoooxoooooooxxxxoxxoxxoxxoxoxxxoxxooooxxxooxxxxxoxxxoxoxooxxooxoxxoxooxoxoxxooxxoxooxoxxxoxo
oxoxxooxxoxxoxxoxoxoooxoooxxxxoxxxxooxxxxoxoxoxxoooxoooxxooooxoxxxooxxooxxxxoxxoxxxoxxxxoxxxooxoxxxx
xoxoooxxxoxxxxoxxooxxxxooooxxxoooooxoxxooooxxxxxxoxooooxxooxooxxoxooxoxxxxxxoxxooxoooxxxxxxoxxooxooo
oxooooxxxxxxoxooxooxoxxoxooooooxooxxoooxxxxxxxxoxxxxxxxxxxxooooxxxooxxooxxxoooooxoxxxxxooxoxxooxxxxx
oxxxxxoooooooxxooxxooxxoooooxoxoxxooooxoxxxoooxxooxooxooxxooxooooooxooooxoxoooxooxxoxxxooxoxooxxxoxo
xoxxxooxxxxxxxxoxooooxxoxxxooxoxooxoxxxxxoxxxooxooxxoxoxoxooxoxoooxoooxxooxxooxxxoxxxxxxxooxxoooxoxo
xxoxoxoxxxoxxxoooxxxxxoxoxxxxoxxxoxxoxoxxoxooxxxoxooxxooxxoxxoooxoxoxxxoooxooooxoxoxoooxxxxxoxxxooox
xxoxoxxxxxxooooxxxoxoxxooxooxooxxoooooxxoxoxxoxxxooooxoooxxoooooxxxxxxxxooxoxoxxxxoxxxoxoxxxxxooxxxx
xooooooxoxoxxxxxoxoooxxoxooxxxxxoooooooxxxxoxxxooxxoxxxooxooxooxxooooxxxxoxxoxxoxoxxoxxxxoooxxoxxxxx
xxooooooxxoxoxxxxxoxxooxoooooxxxxoxoxooxoxooooxxooxxooxooxoxxoooxxoxoxoxxxxoxxxoooooooxxoxooooxxoxxx
xxxxxxoxxxxooxxoooxooooxxoxooooxoxxxxoooooxoxoxxxxoxoxoxxxoxxoxoxoxxxoxoxxxoooxoxooooxoxxxooxoxoooxo
xxxxxxoooxxxoxxxooxxxxoxxxoxoxxoooxxoooxooxoxoxxooxxxxxooxooxoooxoxoxoxooooxoxxoxooooxxoxoooxxxxoxxx
xooxxxxooooxxoxxxooxoxxxxooxxxxxoxooxxoxoxxoooxooxxoxxxooxxoxxxooxxoooxxxooxxooxxxooxoxxooxxxoxxoooo
xxxoooxxxooxxooxoooooxxxxooxoxoxxxxxooxxoooxxooxxooxoxxoxooooxooooxxooooxxxxxxxxxxxoooxoooooooxooxxo
oxxoxxxxxooxoxxxxxooooxxoooooxoooxoxoxxooooxooxxooxxoxxoooxoxoxoxooxooxxxxoxxxoooooxxoooxoxxxooxxxox
xoxooooxoxooxoxoxoxxxooxxxxxxxxooxoxxxoxooooxxoxooxoxoxoxoxoooooooxxxoxoooxxooxxxxooxooxxoxxooooxxxx
xoooooxxxoooxooxxoxxoxxoxoxooooxxoxooxoxxxoooxxxxxxoxoxxoxooxxoxxxoxxxoxooxooxooxoxooooooxooooxooxox
xooxxoxooooxxxooxxxxxoxoxxoxxoxoxxxooxxxxooxoooxoxooxxxooxoooooxoxxxxxooxoooooxxxoxoxxxxxoxxxoooooxx
xooxoxoxoxxooxxoxoxxxxxxxxxxxooxxxxoxxxxxoxxoxxoooxxoxxxxxxoxooooxoxxoxoxooxxoxoxoxxxooxoxxxxxxxxxoo
oxxooxooxxxxxxxxoooxxxoooxxoxxxxoxoxoxoxxooxxxxoxoooxxooxoxoxxxooooooxoooxxxoooxoxoxxoxxoxoxxoxoxxxo
xooxooooxxoxxxooxxooxooxooooxooxxooxxxxooxooooxxxxooxooxoooooxxoxxoxooxxxoxooooxooxxooxoxoxoxxxoooox
ooxoxxxxxooxxoxxxoxooooxxxxxooxxxxxoooooxxoxoxxoxooooooooooxxxxoxoxxxxooxoxxxoxxoxooxoxxoxxxxxoxxxox
xxoxoxooxooxoooooooooxxxxxxxoooooxxoooxxxxxxxxxxxxxooxxxoxxxoxoxoxxxooxoxoxxooxooxoxxxxxoxooooxxoxxo
xxoxoxxxooxoxxooooxxoxxxoxxxxooxxooxoooooooxooooxooxxxoooxoxxxoooooxxoxooooxxooxoxooxxxoxoxxooxxoxxo
ooooooxoxxoxxooooooxoxxoxxoxoxxxoxxxxoxxoxxxooxoxxxxoooooxxoxxxxxxxxxoxxoxooxxxooooxoxoooxooxxxxxoxo
xoooooxooxooxxooxxoxoxxoxxxxxoxoooooxooxoxxooxxooxxxxxoxxxoxooxooxooxoooooooxoxxxoxoooxxxxxooxoxxoxx
xxooooxooxxoxooxoxxxxxoxxxoooxxxxxxxoooxoxoooxxxooooxoxooxoxxooxxooxxoxxxooxoxxooxxoxooxoxxooooxoxox
oooxoooxoxooooxxooxxxxxoxooxxoooxxoxxoxxxoooxxoxoxoxxooxooxxxxoooooxoxooxoooxooxoxoxxxxoxoxxooxxooox
xxooooxoxxxoooxxoxxxxxxxoxxooxoxooooxxxooxoooxxooooxooooxooooxxxxxooxxoooxooxoxxxooxxxooooxxooxoxxxo
ooxoxxoooooooxooxoxxoxxxxxxoooxxxoxxoooooooxooxxooxoxooxoxooxoxoooooooooxxoxxoxxooxooxoxxoooooxxxxox
xxxoxooxxxoooooxxoooooxoxoxoxxooxoxooooxxoxxoooooxxxooooooooxxxoxoxxxoxxoxxxoxoxooxxxxxoxoxoxoxxxxxx
xoooooooooxooxoxoxooooxxxooooxxoooxxxooooxxoxxxooxxxoooooxxoxxxxxxooooooooooooxxxxoxxxooxxxxxoxxxxxx
oxooxooooxooxxxxooxoxxoxxoxooxoxxxooxooooxoxxoxxoxooxooxoxoxooxxxoooxoxooooxoxxxxooxoooxxooxxoxxxooo
ooxoxoxxooxxxoxxxoxxoooooxoooooooxxoxxxxxxxxoxooooxxxxxooooooxxxxooxooxxoooxxoxxoooxxxxxxxoxxxoxoxox
xoxooxooxooxxoxxoxooooxooxxxxoxoxoxxxooooxxoxoxoxoxoxxxooxooooxxooooxoxoxxooxooxxooxoooxxxooxxxxxoxx
xxxoxxxooxoooxxoxxxxoooxoooooxoxoxooooxxoxoxoooxoxxxxxooxooxxxxooxxooooxxxxxooxoxooooxoxxxoooxxxxooo
xxoooxxoxxoxxxxoooxooxxxxxxxooxooooxoxoxxxoooxoxooxxxxxxoxxooxoxooxxxxxxoxxxxoxooxxoxooxoxxoxoxxxxxo
oxxoxooxoooooooxooooxxooxooxoxxxxxooooooxoxxxooxoxoxxxxxxxoxoxxoooooooxoxooxoxxoooxxxoxxooxxxxxxooox
ooxoxooxxooxooxxooxooxoxxxxxooxxxooxooxooxoxoxxxoooxxooxxxxxooxooxxxooxoooxooxoooxxxooxxxxxxooooooxx
xoxxooxooxxooxooxoxoxxoxooxooxooxxooooxxooxxooooxoxoxoooooxxooxoxooooxooxoxoxoxoxoooxxxxooooooxxoxxx
xxooxxxoooxxxooxxooxxxoxxooxoooxooxoxooxoxxxxoxxoxoxxxooxooxoooxxoooooooxxooooxooxxoooxoxoooxxxoooxo
oxoxxxoxxxxooooxxxxxoxooxooooxxxxxooooxoxxoooxooxxooooooooxxoxxxoxooxoooxoxxoxooooxoxxoooxoooxoxxooo
xxoooxoxooxxoxxxoooooxoooxxooooxooxoxxxxoooxoxoxoxooxooxxxxoxxoxxxxoooxoooxxxooxxoxoxxxxooxxooxoooxo
xoooxxxoxoooxxxoxxxxxxxxxoxoxxooxoxxxxoooxxoxoxxxxoooxxxxxxoxoooxxxxxoxxxoxxxooxxxoooooooxxxxxxoxxox
oxooxoooooooxxoxooxxxxxxoxoxxxoooxxooxxxxoxxxoxoxxooxoxoxooooxooxoxxxoxoxxxxxoxoooxxooxxxxxxxxxoxoxx
oooxxxxooooxooxxoxxxxoooooxxxooxxxoooooxoxxxoxooooooxxxoxxoxooooxxoxxxxxoxxxooxxoooxooooxoooooxoxoxx
oxoxoooooxxooxoooxooxxooooxoxxxxxoxooxxoxooooxoxxoooxoxoxoxoxxoxxoxxxooxoooooxoooooxxxoxxoooxoxxoooo
oxxoxooxoooxoxoxooxxoxoxxxxoxxxoooxoxxxoxoooxoxxxooxooooxoooxxoxxxxxoxxoxxoooooooxoooxoxoxooooxoxoox
oooxxoxxxxxoxxxoxooooooxoxoxooxxooxoooxxoxoxxxoxxoooxxxxoxxoxxxoxoxooxxxxooooooxxxooooxoxooxxoxoxxox
xxxooxooxoxxoooxoooxxoooooxxoxoxoxxoxxxxxxxoooxoxxxoooxxxxooxxoxxoxoxooxooxoxxxoxoxxxxoxxxooxxoxxoxo
oxoxooxooxooxxoooxoooxxxoxxxxxxooxoooxoxxxxooooxooxooooxoooxoooxooooxoxxooxxoxoooxxoxooxoxoxxxooxoxx
xxxoxooxooxoxxoxoxxoxxoxxxooxoxoooxooxxooxxoooxxoxooxxxooxooxoooxoooxooxoxooooxxxxxxooooxxoxxoxxxoxx
xoxxxoxoooxxooooxxxoxxooxoxxoxoxoxoooxxxoxxxxxooxooxoooxxooxooxxxxxoxooxoxxxoooxoooxoxoxxoooooxooxoo
oxoxxxoxxxxxoxooooxoxxooxoooxxoxooooxxoxoxoxoxoxxxxooxxxxoxxoxxxoxxoxxooxoooxxooxoxooxxooxxxxxxoxoxx
ooooxxooxoxxoxxxxooxxxoxoxxxoooooxxooxoxooxoooxxooxxxooooxoxoxxoxoooooxoxoxxoxxxoxxxoxoxxooooxxxoxoo
ooxoxxooxxxxxxoooxoooxoxoxoxoxoxxooxoooooxxxoxoxoooxxoxoxoooxxxoxoxxooxxxxooxooxxooxoxoxxooxxxxoxoxx
oooooxoxooxxxoooxxoxooxxxxxxooooxooooooxooxxoxoxoooxxxxxxooxooxoxxxxxxoooxxxxxxoxxoxxxoxoxooxooxoxxx
ooooxxxooxoxxxxxooxxxoxxxxxoxoxxoxoxxoxxoxxoooxoooxoxxoxxxxoxxoxxxoooxoooxoxxoxxooooxxxooxxxoxoxxxoo
ooxxooooxoxxxxoooxoxooxxooxxoxxxoxoxoxxooxxxxxoxxoxoooxxoxoxxxxxxoxxoxoxoxooxxxxoxoxxxxoxxxxxoxxxxxx
oxooxoxxxoxoxxxoooxoxxxoxxxoooxxooxxoxxoxooxxxxxoxxxooxoxoxoxoxooxxooxoooxxxoxoooxooxxxoxxxoxxxoxoxx
xooxxxxoxoxooxooxoxxoxxoooxoooxoooxxoxooxoxxxoxxooxxoxxoxxxxxxxxoooxxooooxooxxoxxxxooxooooxxoxxxxooo
oooooooxxxoxoxxoxxooxxxooxxoxoxoxoxoooxoxxxoxxxoxooooooxooxxoooxoxxoxxooxxxoxooxoxoxxxxxooooxoxooooo
