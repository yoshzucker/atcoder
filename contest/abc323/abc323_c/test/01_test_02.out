2
11
12
7
7
11
7
9
12
3
4
10
8
1
6
5
2
10
10
8
12
1
11
7
6
7
10
4
7
6
8
9
1
7
5
5
12
3
11
13
9
9
3
12
9
12
11
8
9
11
15
8
7
6
5
13
17
7
3
11
12
4
13
7
8
5
6
1
7
5
4
11
7
9
14
6
6
2
4
4
5
15
12
3
0
7
6
11
3
7
5
11
5
7
7
11
14
9
7
3
